assets/js/editor.js

```diff
@@ -331,10 +331,16 @@
   $('.md-editor').each(function() {
     var textarea = this
     var formEditor = $(this).closest('form')
+    const mdeUniqueKey = (window.location.pathname + window.location.search + '@' + this.getAttribute('name')).replace(/[?|&]page=(\d+)/g, '')
+
+    const smdeUniqueContent = localStorage.getItem('smde_' + mdeUniqueKey)
     let minHeight = '500px'
     if ($(this).hasClass('mini-editor')) {
       minHeight = '200px'
     }
+    if (smdeUniqueContent != null) {
+      $('<div class="alert-box"><span>Ce message provient d\'une sauvegarde automatique du contenu. <a onclick="localStorage.removeItem(\'' + 'smde_' + mdeUniqueKey + '\'); window.location.reload(true)" href="javascript:void(0);">Cliquez-ici</a> pour revenir à la version originale.</span><a href="#close-alert-box" class="close-alert-box ico-after cross white">Masquer l\'alerte</a></div>').insertAfter(this)
```

Comment:
```suggestion
      $('<div class="alert-box"><span>Ce message provient d\'une sauvegarde automatique du contenu. <a onclick="window.editors[' + textarea.id + '].value(""); localStorage.removeItem(\'' + 'smde_' + mdeUniqueKey + '\'); window.location.reload(true)" href="javascript:void(0);">Cliquez-ici</a> pour revenir à la version originale.</span><a href="#close-alert-box" class="close-alert-box ico-after cross white">Masquer l\'alerte</a></div>').insertAfter(this)
```

Avec ton code actuel j'ai peur qu'il puisse y avoir plus d'une seconde entre l'appel de `localStorage.removeItem()` et le rechargement de la page, hors avec mes modifications plus haut, l'éditeur sauvegarde le contenu toutes les secondes. Je propose donc de vider l'éditeur avant.

Ajouter un petit `FIXME` en commentaire pour qu'on revienne dessus plus tard et qu'on corrige la forme serait pas mal je trouve.

Reply:
![image](https://user-images.githubusercontent.com/18501150/73867952-e334fd00-4847-11ea-9144-2bff7e39dffd.png)

Je verrais plus le bouton en haut que tout en bas du texte surtout si le message d'origine ou la section fait 210 lignes.

Reply:
Il faudrait remplacer la fin par : `.insertAfter($(this).find('editor-toolbar')).css({margin: '0', padding: '2px inherit'});`